src/dotnet/commands/dotnet-tool/install/LocalizableStrings.resx

```diff
@@ -161,12 +161,19 @@ Tool '{1}' (version '{2}') was successfully installed.</value>
     <value>The settings file in the tool's NuGet package is invalid: {0}</value>
   </data>
   <data name="ToolInstallationFailed" xml:space="preserve">
-    <value>Tool '{0}' failed to install.</value>
+    <value>Tool '{0}' failed to install. This failure may have been caused by:
+
+* You are attempting to install a preview release and did not use the --version option to specify the version.
+* A package by this name was found, but it was not a .NET Core tool.
+* The required NuGet feed cannot be accessed, perhaps because of an Internet connection problem.
+* You mistyped the name of the tool.</value>
   </data>
   <data name="ToolInstallationFailedWithExplicitVersionGuide" xml:space="preserve">
-    <value>Tool '{0}' failed to install. You may need to specify the version using
+    <value>Tool '{0}' failed to install.
```

Comment:
@KathleenDollard @peterhuene this is the message when nuget restore failed

Reply:
To me, the current `ToolInstallationFailed` message used above makes more sense for this use case, which covers restore failures (i.e. prerelease, package type mismatch, nuget feed down, couldn't find package, etc).